src/phoenix/server/api/context.py

```diff
@@ -18,3 +19,4 @@ class Context:
     export_path: Path
     corpus: Optional[Model] = None
     traces: Optional[Traces] = None
+    config: Optional[UMAPParameters] = None
```

Comment:
I think if it's just umap parameters, we should probably name it `umap_parameters` here for clarity. We can then aggregate config(s) in the future under a config object to mount in the server context.